### PR TITLE
Migration fixes

### PR DESCRIPTION
**Pull request by devalih**
https://app.asana.com/0/96288420553298/166207313680254

Now all migrations work like charm from scratch,

# To migrate existing dogwood DB you need to run fake-initial first
```
python manage.py migrate --fake-initial
```

---

# What to Test

 - [x] Migrating from scratch
 - [x] Migrating from an existing production cypress database
 - [x] Migrating from a cypress dev database (optional)

# To migrate cypress database
The steps,

Deploy a machine for each of
cypress-last
dogwood-first
migration-fixes

Run lms/cms migrate on **cypress-last** branch
Run lms/cms migrate --fake-initial on **dogwood-first** branch
Run lms migrate --fake-initial on **migration-fixes** branch (to fake init edraak apps) 
Run lms/cms migrate on **migration-fixes** branch

### CODE REVIEW

File: common/djangoapps/edraak_ratelimit/models.py [hunk 1]
```diff
@@ -37,3 +37,5 @@ class StudentAccountLock(LoginFailures):
 
     class Meta:
         proxy = True
+        managed = False
```
Comment (OmarIthawi) [inline common/djangoapps/edraak_ratelimit/models.py]: Some documentation is required here, we write together something meaningful here.

**Comment by OmarIthawi** [comment]
@devalih and @ahmedaljazzar I've rebased this branch to the latest master, so you might want to force update your local branches accordingly.

**Comment by OmarIthawi** [comment]
@devalih It is still not working from first attempt. You still have to run the migration twice on a clean database.

**Comment by OmarIthawi** [comment]
Migrating from scratch works fine with me so far.

**Comment by OmarIthawi** [comment]
I'm assigning you @ahmedaljazzar because I've partly worked on this pull request. It makes sense to have another one reviewing this.

**Comment by devalih** [comment]
I'm testing Migrating from an existing production cypress database...

**Comment by devalih** [comment]
Migrations works after several tests today on production snabshots

**Comment by ahmedaljazzar** [comment]
LGTM 👍

**Pull request merged**